recipes/rotor/all/conanfile.py

```diff
@@ -30,34 +34,40 @@ class RotorConan(ConanFile):
         "multithreading": True,
     }
 
-    exports_sources = "CMakeLists.txt"
-    generators = "cmake", "cmake_find_package"
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
 
     def requirements(self):
-        self.requires("boost/1.79.0")
+        self.requires("boost/[>=1.68.0]")
```

Comment:
We do not use version ranges in CCI because it makes the builds less deterministic, you can check the faq for more details 

This should just be the newest support version  :)